src/action/ActionCancel.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace TaskForce\action;
+
+class ActionCancel extends Action {
+    public function getActionName(): string {
+        return 'Отменить';
+    }
+
+    public function getInternalName(): string {
+        return 'actionCancel';
+    }
+
+    public static function checkRights(
+        int $userId,
+        int $customerId,
+        ?int $executorId
```

Comment:
Опциональным аргументам лучше сразу давать значение по умолчанию, чтобы их не приходилось передавать напрямую.
Проще написать `ActionCancel::checkRights(1, 2)`, чем
`ActionCancel::checkRights(1, 2, null)`
```suggestion
        ?int $executorId = null
```